docs/ur/glossary.md

```diff
@@ -0,0 +1,94 @@
+# Glossary
+
+| English         | Urdu         |Context/Comments |
+| ------------- |:-------------:|:-------------:|
+| Active/Enable |**فعال**|
+| Addons  |**ایڈ اون**|
+| Advertisement/Ads |**اشتہار**|
+| API  |**API**|
+| Approved |**منظور شدہ**|
+| Articles |**مضمون**|
+| Auto |**خودبخود**|
+| Autoscroll  |**خودبخود سکرال**|
+| Authentication |**توثيق کارى**|
+| Archive |**آرکائیو**|
+| Block |**روکیں**|
+| Bookmark  |**بکمارک**|
+| Browsing |**براؤزنگ**|
+| Browser |**براؤزر**|
+| Bug |**بگ**|
+| Blog |**بلاگ**|
+| Certificate |**تصدیق نامہ**|
+| Check |**پڑتال**|
+| Comment |**تبصرہ**|
+| Clear | **صاف کریں**|
+| Configurations |**تشکیل کاریاں**|
+| Configuration |**تشکیل کاری**|
+| Content |**مواد**|
+| Category |**زمرہ**|
+| Click |**کلک**|
+| Dark |**گہرا**|
+| Data |**ڈیٹا**|
+| Debug |**ڈیبگ**|
+| Debugger | **ڈیبگر**|
+| Delete |**حذف**|
+|Edit |**تدوین**|
+|Editor |**ایڈیٹر**|
+|Erase |**مٹائیں**|
+|End user / User |**صارف**|
+|Extension |**ایکسٹینشن**|
+|Feature|**خصوصیات**|
+|File |**فائل**|
+|Flash plugin|**پلگ ان Flash**|
+|Folder |**فولڈر**|
+|HTTP/HTTPS|**HTTP/HTTPS**|
+|Inactive/Disable |**غیر فعال**|
+|Install |**انسٹال**|
+|Key |**کلید**|
+|Keyboard |**کی بورڈ**|
+|Layout |**لےآؤٹ**|
+|Live |**لائیو**|
+|Manage |**منظم**|
+|Method |**طریقہ**|
+|Location |**مقام**|
+|Current Location|**موجودہ مقام**|
+|Online |**آن لائن**|
+|Pop up|**پوپ اپ**|
+|Rate |**شرح**|
+|Rating |**شرح کاری**|
+|Recommended |**تجویز کردہ/سفارش کردہ**|
+|Remove|**ہٹائیں**|
+|Save |**محفوظ**|
+|Search |**تلاش کریں**|
+|Section |**صیغہ**|
+|Setting |**سیٹنگ**|
+|Settings |**سیٹنگز**|
+|System |**سسٹم**|
+|Sync |**سنک**|
+|Synchronize |**سنکرونیس**|
+|Synchronizing|**سنکرونیزیشن**|
+|Theme|**تھیم**|
+|Toolbar |**ٹولبار**|
+|Toggle |**ٹوگل **|
+|Tracker/Trackers |**ٹریکر**|
+|URI|**URI**|
+|URL|**URL**|
+|Window |**ونڈوں**|Relevant to browser window not OS|
+
+**Brand names to be used as it is:**
```

Comment:
Please refer to this as well here: https://mozilla-l10n.github.io/styleguides/mozilla_general/#brands-copyright-and-trademark

Reply:
Sure.